### DISCUSSION TITLE
Sharing S3 repository cache amoung multiple GitLab Runners

### DISCUSSION THREAD

**Discussion by chezsmithy** [renovatebot/renovate#28452] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

We are looking to implement the s3 backed repository cache feature. We are curious if the s3 location can be shared among multiple GitLab runners each processing a piece of the overall repository list? Are we required to have separate paths per runner for the cache storage or are they able to share a single location?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment, marked as answer]
you can share as long as not two runners are processing the same repo.
